### ISSUE TITLE
AudioRecorder - selectively  display recorder (or hide the audio widget) or changing to a HBox layout

### ISSUE THREAD

**Issue by sathayen** [maartenbreddels/ipywebrtc#85]
I have created an AudioRecorder instance  in a jupyter notebook for recording from the microphone. 

The jupyter notebook output displays a recorder widget and an audio widget. Is there a way to turn off the audio widget display? I could call `recorder.audio` to show the audio player widget only , but could not find a way to only show the `recorder` widget. 

I tried: 
`recorder.audio.layout.display = 'none'`  and then invoked the `recorder` object. But it still displays the audio widget. 

If I set `recorder.layout.display` to 'none' then it hides all the widgets. Is there any attribute that will only display the 'record' widget and not the audio? If not, how can this be accomplished?

**Comment by martinRenou** [comment]
There is currently no way of doing this properly unfortunately.

**Comment by sathayen** [comment]
thanks @martinRenou , could you please suggest any hack to get this working? Ideally, I want to be able to put the two widgets in a HBox layout like so : `widgets.HBox(recorder, audio)` .. is there a woraround to accomplish that (if not hide the audio widget)

**Comment by martinRenou** [comment]
Yeah you could hack around it by injecting the right CSS in the page, that's not a great solution but it would work.

```python
from IPython.display import HTML

display(HTML("""
<style>
.jupyter-widgets audio:nth-child(2) {
  display: none;
}
</style>
"""))
```

This is really not a good solution, but it would work.

**Comment by sathayen** [comment]
That works for now! thanks! :)
